infrastructure/main.tf

```diff
@@ -15,6 +15,9 @@ locals {
   cors_origin = "${var.cors_origin != "" ? var.cors_origin : local.default_cors_origin}"
   ccd_print_service_url = "http://ccd-case-print-service-${local.env_ase_url}"
   document_management_url = "${var.document_management_url != "" ? var.document_management_url : local.default_document_management_url}"
+
+  // S2S
+  s2s_url = "http://rpe-service-auth-provider-${local.env_ase_url}"
```

Comment:
i would think you'll have preview env build failures on this, but you would also have that problem here for all of your urls